jobboard/templates/jobboard/job.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'base.html' %} {% block content %}
+
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-md p-5 text-align">
+            <h2>{{ job_data.title }} </h2>
+            <h6> Company: {{ job_data.company }}, in {{ job_data.city }}</h6>
+            <h7>{{ job_data.job_type }} job ;</h7>
+            <h7> Work from: {{ job_data.work_from }}</h7>
+            <br> <br>
+            <h6><strong>{{ job_data.description }}</strong></h6>
+            <br>
```

Comment:
3 nested `if`s without tests...

Reply:
will add tests for it

Reply:
@ifireball Should I add separate tests for different users for this one?

Reply:
@ifireball Should I add separate tests for different users for this one?

Reply:
@ifireball I checked the content of the `job_data` , `is_student` and `is_recruiter`  keys in the dictionary in each of the tests.

how else can I check those ifs?

Reply:
I think what @ifireball means is that you should take the logic out of the template.
You can do something similar to what @eldaraharonson did with the board view on 'jobboard/views.py', where he created functions that set the content in the context dictionary according to the user who sent the request.
Then you can test it by checking that the correct data is passed through the context.
You can also look at what I did with the nav bar links it follows the same idea

Reply:
Eldar did the exact same thing as I did, just that he made separate functions that insert the content to dictionary. and he used the `user_is_student` and `user_is_recruiter` in his html page. And - I did exactly what you did, same functions implementation, just inside the `detail_view` function.
I check in each one of the tests these fields. I think it's exactly the same, and I think I checked all these fields

Reply:
Yea actually you're right Eldar did the same thing.
What I did with the nav bar is handling all the `ifs` outside but I'm not sure how and if you can do that here as in one case you have to display a button and another case a paragraph.
Anyone else has an idea?

Reply:
Here is one way to to it that I described during the demo on Sunday:

In the view code:
```python
if Student.is_student(request.user.id):
    user_indicator_template = "student_user_indicator.html"
elif Recruiter.is_recruiter(request.user.id):
    user_indicator_template = "requiter_user_indicator.html"
else:
    user_indicator_template = "other_user_indicator.html"
context["user_indicator_template"] = user_indicator_template
```

In the template code:
```jinja2
{% include user_indicator_template %}
```

Reply:
@ifireball  fixed that. Please take a look

Reply:
looks good yael